gozo-stores/src/context/cart-context.js

```diff
@@ -0,0 +1,68 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios"
+
+const CartContext = createContext();
+
+function CartProvider({children}) {
+
+    const [myCart, setMyCart] = useState([])
+
+    const addToCart = async (token, product) => {
+      try {
+        const res = await axios.post('/api/user/cart', {product}, {
```

Comment:
=>try to use proper names res i assume is for response so be it response.
=>you can destructure the response here directly const {data : { cart}}

Reply:
I'll take care of both things